mongodbatlas/resource_project_api_key.go

```diff
@@ -30,8 +30,9 @@ func ResourceProjectAPIKey() *schema.Resource {
 		},
 		Schema: map[string]*schema.Schema{
 			"project_id": {
-				Type:     schema.TypeString,
-				Optional: true,
+				Type:       schema.TypeString,
+				Optional:   true,
+				Deprecated: fmt.Sprintf(DeprecationByVersionMessageParameter, "1.16.0"),
```

Comment:
take a look at the internal/common/constant package

Reply:
thanks!